### DISCUSSION TITLE
Allow users to remove themselves from the waiting list

### DISCUSSION THREAD

**Discussion by vibe123456** [pretix/pretix#1970] [Ideas]
If one cannot buy a ticket because there are no more tickets left, he can subscribe to the waiting list. It would be nice if you could unsubscribe from the waiting list too. 
Another improvment would be: 
Allowing to add more fields for subscribing to the waiting list, like "Name" "Telefon number" etc.
Maybe you could implement the waiting list as another state of the costumer? (like paid, unpaid, overdue etc.)?

**Comment by raphaelm** [comment]
> If one cannot buy a ticket because there are no more tickets left, he can subscribe to the waiting list. It would be nice if you could unsubscribe from the waiting list too.

Sounds useful (but probably a lot of work for a small feature).

> Allowing to add more fields for subscribing to the waiting list, like "Name" "Telefon number" etc.

Hmm, what would be the use case for that?

> Maybe you could implement the waiting list as another state of the costumer? (like paid, unpaid, overdue etc.)?

No, that doesn't fit pretix' data model at all, sorry.

**Comment by rimi-itk** [comment]
We've just had a similar request:

In the mail sent to a customer on the waiting list, our client wants to include a ”Not interested anymore”-link that the customer can use to give up his entry on the waiting list (without buying a ticket).

@raphaelm, do you have an idea of how much ”a lot of work” is to implement something like this? 

Regards,
Mikkel

**Comment by raphaelm** [comment]
> In the mail sent to a customer on the waiting list, our client wants to include a ”Not interested anymore”-link that the customer can use to give up his entry on the waiting list (without buying a ticket).

This seems like a different request to me, @David100mark proposed an "remove me" option *while still waiting*, whereas your request is for an "remove me" link *after being selected*. Both seem useful to me, the latter seems pretty easy to implement to, we just need to drop a link in the email and have a "give up a waiting list voucher" endpoint, I'd estimate this for ~2 hours of work.
